.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: cubyn/node:ci-7
+      - image: cubyn/node:ci-8.6
```

Comment:
[Why 8.6](https://node.green/#ES2018-features-object-rest-spread-properties-object-spread-properties)
Seems safe to bump